dts/arm/st/wb0/stm32wb0.dtsi

```diff
@@ -217,6 +217,31 @@
 			interrupts = <7 0>;
 			status = "disabled";
 		};
+
+		dma1: dma@48700000 {
+			compatible = "st,stm32-dma-v2bis";
+			#dma-cells = <2>;
+			reg = <0x48700000 256>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB0 (1 << 0)>;
```

Comment:
Now that #79683 has been merged, this node should be updated

Reply:
Done + added a commit to update nodes missed by the PR (I2C/SPI)